src/handlers/swap_handlers.ts

```diff
@@ -76,6 +76,14 @@ export class SwapHandlers {
             throw new InternalServerError(e.message);
         }
     }
+    // tslint:disable-next-line:prefer-function-over-method
+    public async getSwapTokensAsync(_req: express.Request, res: express.Response): Promise<void> {
+        const tokens = TokenMetadatasForChains.map(tm => ({
+            symbol: tm.symbol,
+            address: tm.tokenAddresses[CHAIN_ID],
+        }));
+        res.status(HttpStatus.OK).send(tokens);
+    }
```

Comment:
Just sending an array in a JSON response can mean trouble later on. Can we return this like 
```
{ tokens: [] }
```

Reply:
ahh you mean in future changes to the endpoint in that it is difficult to make non-breaking additions?